Gemfile.lock

```diff
@@ -125,10 +125,10 @@ GEM
       parser (>= 3.1.1.0)
     ruby-progressbar (1.11.0)
     ruby2_keywords (0.0.5)
-    sentry-ruby (5.3.0)
+    sentry-ruby (5.3.1)
```

Comment:
Could this not be 5.3.4 like the others? That would remove the `sentry-ruby-core` dependency too

Reply:
Good catch, odd that form-api pins gem versions but the other two apps don't